README.md

```diff
@@ -170,3 +170,53 @@ Use the `Makefile`. Run without any arguments for more information:
 ```
 make
 ```
+
+### Smoke test
+
+This will run a basic smoke test against a deployed environment.
+
+#### Dependencies
+
+Based on ruby 2.2.2, it is recommended to use `rbenv` or `rvm`.
+
+Install the dependencies:
+
+```bash
+bundle install
+```
+
+#### Run
+
+You must pass the `TSURU_USER` and `TSURU_PASS` environment variables, and
+optionally the `DEPLOY_ENV` (defaults to `ci` if missing).
+
```

Comment:
Is is not necessary to specify TSURU_USER/PASS on as environment variables since you're pulling them out of the vault now... Docs should be updated.